### ISSUE TITLE
radar data seems to not have velocities in it

### ISSUE THREAD

**Issue by SM1991CODES** [leddartech/pioneer.das.api#8]
Hi,

I am trying to get a point cloud out of the radar data.
However, I face the following problems:
1) The point cloud for radar is (Nx3), where z = 0 always. So, is it not 3D?
2) How do I get velocities for each point?

Also, please help with the documentation. the project seems very hard to follow along. Is there any better documentation of example how to use it to get frame by frame data and annotations for each sensor?

Best Regards
Sambit

**Comment by jeanlucdeziel-leddartech** [comment]
Hi,

Yes, the radar data is quite sparse compared to the lidar data.

For extrinsics:

The path to the extrinsics directory is not provided in the configuration file (platform.yml) for the radar. It seems to be an error on our part, I will pass the information. In the meantime, you can add the path yourself. To do so, add:
  extrinsics: *extrinsics_folder
under the line "radarTI_bfc:" in platform.yml. If you use multiple datasets, you need to do that for every platform.yml. I suggess you automate that with a script because there are a hundred of them.

For egomotion compensation

2. There is a error in the API with regards to the radar timestamps, so calling "radar_sample.point_cloud(undistort=True)" results in an error. I found a fix, however, it would not help because the radar doesn't have individual timestamps for each detection, which is necessary to perform motion compensation.

Hope this helps
